server/server/server.go

```diff
@@ -209,6 +215,23 @@ func (s *Server) Start() error {
 		s.telemetry.RunTelemetryJob(firstRun)
 	}
 
+	var group run.Group
+	if s.config.PrometheusAddress != "" {
```

Comment:
We should check for `s.config.Telemetry` as well since otherwise we'll not be recording metrices.

Reply:
could you elaborate? why telemetry is related here?

Reply:
Information sent to Prometheus is about customer's configuration, so I think falls under the tag of tracking metric. We do the same in Mattermost.

Reply:
good point though prometheus is about operating focalboard right? Telemetry is more analytics oriented right?  @sbishel thoughts?

Reply:
up to you guys how we can treat this to avoid confusion for the customer.

Reply:
I think telemetry and metrics are separate things. Metrics are about performance and telemetry is more about configuration settings. We probably also want the ability to "turn off" metrics. (ie. "enableMetrics") separately from the "PrometheusAddress". Also we should give this a generic name rather than "Prometheus".,
https://docs.mattermost.com/deployment/metrics.html
https://docs.mattermost.com/administration/telemetry.html

Reply:
agree. it can be `EnableMetrics`

Reply:
@sbishel @harshilsharma63 let me know if we will rename it so I can quickly commit the change.